purpl3_rms/src/App.js

```diff
@@ -20,90 +19,106 @@ import ErrorDoc from './components/errorDoc/errorDoc'
 import ComputerPage from './components/computers/computers'
 import PrivateRoute from './components/PrivateRoute'
 import {LinkContainer} from "react-router-bootstrap"
+import { logoutRequest } from './libpurpl3/purpl3API';
+import {ErrorModal} from "./components/modals/errorModal"
+import {ErrorContext} from "./context/errorContext";
+
 
 function App() {
 
   const [loggedIn, setLogin] = useState(false);
+  const [error, setError] = useState(null);
 
   return (
     <div className="App">
+      <ErrorContext.Provider value={[error, setError]}>
```

Comment:
wrap it in this ErrorContent.Provider thingy?

Reply:
This is used so Error is global so we only need to have one error modal instead of a bunch ie only one error can show at a time